docs/install/Knative-with-Gloo.md

```diff
@@ -82,7 +82,7 @@ Now you can deploy an app using your freshly installed Knative environment.
 ## Configuring DNS
 
 Knative dispatches to different services based on their hostname, so it greatly
-simplifies things to have DNS properly configured. For this, we must look up the
+simplify things to have DNS properly configured. For this, we must look up the
```

Comment:
the previous wording was correct; please leave "simplifies"

Reply:
done